### DISCUSSION TITLE
urm, whan do I do her

### DISCUSSION THREAD

**Discussion by RedMan13** [PenguinMod/PenguinMod-Home#81] [Q&A] [answered]
q: where did my mom
a: at ma house

q: wth is "menguinpod"
a: what are you talking about????

q: why did you make this
@: idk, cause it's funny og?

q: can I have your mother since you have mine?
a: sure, I can give you "your mother" if you pay me 18q pennies

q: wth is 18q pennies
a: uhhhhhh about 20 30's

q: what
a: huh

q: why is this still going
a: nuh uh

**Comment by derpygamer2142** [comment, marked as answer]
yes, but i would recommend consulting an accountant first

**Reply by RedMan13** [reply]
ah, alr
